### PR TITLE
feat(Pipeline Migration to 1ES Template):accessibility-insights-windows

### PR DESCRIPTION
**Pull request by JeevaniChinthala**
#### Details

Migrated accessibility-insights-windows to 1ES template as per the changes mentioned in the documentation link provided for the 1ES pipeline migration.

##### Motivation

Pipeline migration to 1ES template. User Stories - [User Story 2136511](https://dev.azure.com/mseng/1ES/_workitems/edit/2136511): Migrate accessibility-insights-windows-CI to 1ES template and [User Story 2136512](https://dev.azure.com/mseng/1ES/_workitems/edit/2136512): Migrate  accessibility-insights-windows-PR to 1ES template
Old Pipeline for CI: https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/results?buildId=51302&view=results
New Pipeline for CI: https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/results?buildId=51378&view=results
Old Pipeline for PR: https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/results?buildId=51301&view=results
New Pipeline for PR: https://dev.azure.com/accessibility-insights/Accessibility%20Insights/_build/results?buildId=51377&view=results

#### Pull request checklist
Addresses an existing issue: User Stories - [User Story 2136511](https://dev.azure.com/mseng/1ES/_workitems/edit/2136511) and [User Story 2136512](https://dev.azure.com/mseng/1ES/_workitems/edit/2136512)

- [x] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/main/docs/Scenarios.md) completed?
- [x] Does this address an existing issue? [User Story 2136511](https://dev.azure.com/mseng/1ES/_workitems/edit/2136511) and [User Story 2136512](https://dev.azure.com/mseng/1ES/_workitems/edit/2136512)
- [na] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [na] Attach any screenshots / GIF's that are applicable.
 
##[warning]1ES PT Warning: Failed to fetch default repository branch. 
This is an existing issue. Please refer
[Bug 2119901](https://dev.azure.com/mseng/1ES/_workitems/edit/2119901): Address issue "Failed to fetch default branch"

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: build/build-and-unit-tests.yml
Comment (DaveTryon) [inline build/build-and-unit-tests.yml:10]: Not sure why, but the DisplayName is getting ignored in the pipeline output

**Comment by DaveTryon** [comment]
Is there a way to disable BinSkim in these pipelines? It doesn't appear to be configured, and since we don't sign anything in this pipeline, it's more or less guaranteed to fail if we scanned the files that we build

File: build/build-and-unit-tests.yml
Comment (JeevaniChinthala) [reply build/build-and-unit-tests.yml:10]: Updated the new pipeline link.

File: build/build-and-unit-tests.yml
Comment (DaveTryon) [inline build/build-and-unit-tests.yml:13]: What's the intended indent on the outputs section? It seems far more indented that I would expect

**Review by DaveTryon — APPROVED** [review]
@JeevaniChinthala, the pipelines look great. I think there's a case of indenting problem, but I'm approving to unblock you. No need to get another approval after the indents are updated.

File: build/build-and-unit-tests.yml
Comment (v-viyada) [reply build/build-and-unit-tests.yml]: Fixed indentation

**Pull request merged**
